camel-sap/org.fusesource.camel.component.sap/src/org/fusesource/camel/component/sap/util/Util.java

```diff
@@ -616,9 +616,13 @@ public static void reattachStaticPackageSuperTypes(EPackage ePackage) {
 		}
 	}
 
-	private static void ensureXMLPackages() {
```

Comment:
This method seems to be used in `save` and `load` methods (in `org.fusesource.camel.component.sap.util.Util`), perhaps those need to be refactored to use `ensureBasePackages`?